src/components/Shell/ShellAppBar.tsx

```diff
@@ -1,13 +1,13 @@
 import { styled } from '@mui/material/styles'
 import MuiAppBar, { AppBarProps as MuiAppBarProps } from '@mui/material/AppBar'
-import Toolbar from '@mui/material/Toolbar'
-import Typography from '@mui/material/Typography'
-import StepIcon from '@mui/material/StepIcon'
-import Tooltip from '@mui/material/Tooltip'
-import IconButton from '@mui/material/IconButton'
-import MenuIcon from '@mui/icons-material/Menu'
```

Comment:
Why did we switch from default to named imports here? [At least for development, it seems that using default imports is faster](https://mui.com/material-ui/guides/minimizing-bundle-size/#development-environment).